netbox/core/api/schema.py

```diff
@@ -128,6 +159,35 @@ def get_response_serializers(self) -> typing.Any:
 
         return response_serializers
 
+    def _get_request_for_media_type(self, serializer, direction):
```

Comment:
To preserve the default
```suggestion
    def _get_request_for_media_type(self, serializer, direction='request'):
```